client/src/modules/stock/lots/modals/duplicates.modal.html

```diff
@@ -12,6 +12,9 @@
   </div>
 
   <div class="modal-body">
+    <div style="margin-bottom: 0.5em">
+      <span class="text-bold">{{$ctrl.selectedLot.inventory_text}}</span>
```

Comment:
This may seem nit-picky, but some of our users are more comfortable with the inventory code.  Could you add that before the text, separated by a hyphen?
```html
{{ inventory_code}} - {{inventory_text.}}

```

Reply:
Done.